src/main/java/com/odysseusinc/arachne/datanode/config/HandlebarsConfig.java

```diff
@@ -34,6 +34,8 @@
 public class HandlebarsConfig {
 
     private static final String ESTIMATION_RUNNER_TEMPLATE = "estimation/runner.mustache";
+    private static final String PATIENT_LEVEL_PREDICTION_RUNNER_TEMPLATE = "plp/main.r";
+    private static final String COHORT_CHARACTERIZATION_RUNNER_TEMPLATE = "cc/cc_runner.mustache";
```

Comment:
Let's be consistent and use single convention for naming - either "main.r" (which seems more clear for me) or *_runner